.github/workflows/landing-builds.yml

```diff
@@ -378,3 +378,8 @@ jobs:
                             --build-arg DAOS_BUILD_TYPE=dev
                             --build-arg COMPILER=gcc
                             --tag build-image
+      - name: Run NLT
+        if: matrix.distro != 'alma.9'
+        run: docker run --mount type=tmpfs,destination=/mnt/daos_0,tmpfs-mode=1777 --user root:root
+                 build-image ./daos/utils/node_local_test.py --no-root
```

Comment:
'build-image' is the name of a docker image, you have two of the builds above which create images with this name on L339 and L380, I don't know what docker will do about that but in order to run NLT you also need the DAOS_KEEP_SRC option which isn't set for the case on L380.

Delete L390 and I think this will work, it'll run NLT on clang with a non-dev/debug build.